scrapers/nv/events.py

```diff
@@ -1,82 +1,167 @@
+        cur_row = []
+        # Each meeting has info spread across up to 3 divs, with a <hr> after
+        # each section - The following code finds all of these sections with 3
+        # divs and places them into the meetings variable
+        for i in self.root.getchildren():
+            if i.tag == "div":
+                cur_row.append(i)
+            elif i.tag == "hr":
+                # Only add if all three elements are present - missing elements
+                # indicate that this section is cancelled or floor meeting
+                if len(cur_row) == 3:
+                    meetings.append(cur_row)
+                cur_row = []
+
+        for i in meetings:
+            # Second element contains: a link the the agenda pdf
+            # Grabbing the second element first because the start time from the
+            # agenda pdf may be needed
+            agenda = XPath(
+                './/a[@title="View the agenda for this meeting"]/@href'
+            ).match_one(i[1])
+
+            # First element contains: the title and date
+            title = XPath(".//h3/a[1]/text()").match_one(i[0])
+
+            # Extract all committees involved in meeting from the title
+            # Some joint meetings don't start with the joint prefix so extra logic
+            # is needed to get the names of all involved committees
+            add_joint_prefix = False
+            joint_prefix = "Joint Meeting of the "
+            separator = "%$%$%"
+            if title.startswith(joint_prefix):
+                title = title[len(joint_prefix) :]
+                add_joint_prefix = True
+            title = title.replace(
+                " and Senate Committee", f"{separator}Senate Committee"
+            )
+            title = title.replace(
+                " and Assembly Committee", f"{separator}Assembly Committee"
+            )
+            committees = title.split(separator)
+            title = " and ".join(committees)
+            if add_joint_prefix:
+                title = f"{joint_prefix}{title}"
+
+            # Get start date
+            date = XPath(".//h3/a[2]/text()").match_one(i[0])
+            # Remove day of the week information
+            date = date.split(", ", 1)[1]
+            if "[" in date:
+                # Date contains "upon adjourment" instead of start time
+                date = date.split("[")[0]
+                # Check PDF for start time
+                agenda_start_time = next(AgendaStartTime(source=agenda).do_scrape())
+                date = f"{date}{agenda_start_time}"
+
+            date = datetime.datetime.strptime(date, "%B %d, %Y %I:%M %p")
+
+            # Third element contains: a list of locations
+            locations = XPath(".//li/text()").match(i[2])
+
+            # Build event, scrape bill ids from agenda, and yield it
+            event = Event(
+                start_date=self._TZ.localize(date),
+                name=title,
+                location_name=locations[0],
```

Comment:
Can you add a replace statement to remove the no-break-space that is currently showing up in the location name between "Carson" and "City" on any events? `"\u00a0"` needs to be replaced with single space `" "`.